src/index.js

```diff
@@ -51,6 +49,7 @@ class Bitswap {
   _receiveMessage (peerId, incoming, callback) {
     this.engine.messageReceived(peerId, incoming, (err) => {
       if (err) {
+        // TODO: Q: why do we just log and not return here?
         log('failed to receive message', incoming)
       }
```

Comment:
- [ ] Understand why this Error is just logged and why the rest of the function continues.